minio/mcs
# Added Service Accounts page to settings

fixes #53 

## What does this do?

Added service accounts page to settings module

## How to test

- Create an account with permissions to see service accounts or add this fragment of code into line 350 of Console.tsx:

```
<Route
                    exact
                    path="/service_accounts"
                    component={ServiceAccounts}
                  />
```

- Go into Configurations Lists > service_accounts > Edit Icon
- Click on create service account & write a valid policy
- Click on create button, the policy should appear in the list
- Click on the delete button of the recently created policy, the policy should be deleted

![Screen Shot 2020-05-18 at 11 37 52 PM](https://user-images.githubusercontent.com/33497058/82289042-79111580-9969-11ea-8e60-0094b3c56c02.png)
![Screen Shot 2020-05-18 at 11 37 34 PM](https://user-images.githubusercontent.com/33497058/82289044-7a424280-9969-11ea-9464-693ed6ce5cc4.png)
![Screen Shot 2020-05-18 at 11 37 28 PM](https://user-images.githubusercontent.com/33497058/82289047-7adad900-9969-11ea-9596-5a31ce300860.png)
![Screen Shot 2020-05-18 at 11 37 15 PM](https://user-images.githubusercontent.com/33497058/82289048-7c0c0600-9969-11ea-8de8-c247d62f87a7.png)
